runtime/gc_base/ReferenceChainWalker.cpp

```diff
@@ -355,6 +355,11 @@ MM_ReferenceChainWalker::scanObject(J9Object *objectPtr)
 	case GC_ObjectModel::SCAN_CLASSLOADER_OBJECT:
 		scanMixedObject(objectPtr);
 		break;
+#if JAVA_SPEC_VERSION >= 19
+	case GC_ObjectModel::SCAN_CONTINUATION_OBJECT:
+		scanMixedObject(objectPtr);
+		break;
+#endif /* JAVA_SPEC_VERSION >= 19 */
```

Comment:
Should this case be separate or can it be joined to the group above?

Reply:
same reason as above for keeping the separated code.